### DISCUSSION TITLE
Table column widths are not working without a non-resizable column

### DISCUSSION THREAD

**Discussion by madaraszistvan** [TanStack/table#2190] [General]
By removing the column with the fixed width (`disableResizing: false`) from the example with [Full Width Resizable Table](https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/full-width-resizable-table?file=/src/App.js:3093-3108), the widths set in the column config are not working.

Sandbox: https://codesandbox.io/s/tender-poincare-ywogt?file=/src/App.js

Device: MacOS, Browser: Chrome

**Comment by tannerlinsley** [comment]
I'm not seeing how to replicate this with that codesandobx
